src/dbos.ts

```diff
@@ -641,7 +652,16 @@ export class DBOS {
   }
 
   // sql session (various forms)
-  /** @returns the current SQL client; only allowed within `@DBOS.transaction` functions */
+  /**
+   * @returns the current SQL client; only allowed within `@DBOS.transaction` functions
+   * @deprecated - use data source packages such as:
+   *  `@dbos-inc/drizzle-datasource`
+   *  `@dbos-inc/knex-datasource`
+   *  `@dbos-inc/nodepg-datasource`
+   *  `@dbos-inc/postgres-datasource`
```

Comment:
nit: we never supported `postgres` package in DBOS core. We supported `pg` package (also known as node-postgres - but not to be confused with the `node-postgres` or `node-pg` packages )